.github/ISSUE_TEMPLATE/membership.yml

```diff
@@ -0,0 +1,42 @@
+name: Organization Membership Request
+description: Request membership in the CRI-O organization
+labels: [ "area/github-membership" ]
+title: "REQUEST: New membership for <your-GH-handle>"
+body:
+- id: github
+  type: input
+  attributes:
+    label: GitHub Username
+    placeholder: e.g. @example_user
+  validations:
+    required: true
+- id: requirements
+  type: checkboxes
+  attributes:
+    label: Requirements
+    options:
+    - label: I have reviewed the [community membership guidelines](https://git.k8s.io/community/community-membership.md)
+      required: true
+    - label: I have [enabled 2FA on my GitHub account](https://github.com/settings/security)
+      required: true
+    - label: I have one sponsor from the list of approvers in at least one OWNERS file within one of the CRI-O repository
+      required: true
+    - label: I have spoken to my sponsor ahead of this application, and they have agreed to sponsor my application
```

Comment:
Nit:
```suggestion
    - label: I have spoken to my sponsor ahead of this application, and he has agreed to sponsor my application
```

Reply:
please my friend, allow me to introduce you to the [singular they](https://en.wikipedia.org/wiki/Singular_they), a gender neutral alternative to gendered pronouns. It's become the idiomatic way to have documentation be non-gendered and more inclusive :slightly_smiling_face:

Reply:
ups, shame upon me 😅!

Reply:
no shame, only a teachable moment :smile: